### PR TITLE
Meeting minutes

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Write up the meeting minutes from last week's HSF meetings, plus also make an HSF website version of the minutes from the training meeting that were only posted as PDF to that agenda (this is then an examplar version for the organisers).

### CODE REVIEW

File: organization/_posts/2018/2018-04-23-training.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+layout: default
+---
+
+# {{page.title}}
+
+#### Attendees: Albert Puig Navarro (LHCb) , Dario Berzano (ALICE), Dario Menasce (CMS), David Lange (CMS), Francesco Giacomini (Bertinoro School), Ilya (Belle II), Louise Heelan (ATLAS), Markus Schulz (CERN IT), Martin Ritter (Belle II), Rob Kutschke (Mu2e/Neutrino FNAL), Servesh Muralidharan (CERN IT), Sudhir Malik (CMS)
+
+- Attendees introduced themselves as it is big spectrum of people
+- A bit about attendees role in their respective experiments/labs/
+    - Albert Puig Navarro and Dario Berzano (Coordinate StarteKit at LHCb and ALICE)
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-04-23-training.md]: StarterKit

File: organization/_posts/2018/2018-07-19-coordination.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+---
+title: "HSF Weekly Meeting #142, 18 July, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+#### *Present/Contributors*: Graeme Stewart, Andrea Valassi, Dario Menasce, Benedikt Hegner
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-07-19-coordination.md]: Add "Eduardo Rodrigues". There is probably 1-2 persons missing, connecting from outside.

File: organization/_posts/2018/2018-07-18-softwareforum.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Introduction
+============
+-   Next meetings being planned, 
+    [check Indico for the status](https://indico.cern.ch/category/10392/). Please 
+    send any suggestions for presentations or discussion topics.
+
+VecCore
+=======
+-   Why does CPU go down between 2027-2030 in the CMS plot? A.
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply organization/_posts/2018/2018-07-18-softwareforum.md]: It was "A." for "Answer"! Maybe not clear though.

**Comment by graeme-a-stewart** [comment]
OK, no more attendees volunteering, so merging this now.

**Pull request merged**